frontend-instructor/src/main.jsx

```
@@ -40,6 +41,10 @@ const routes = createBrowserRouter([
       {
         path: '/analytics/:courseId',
         element: <CourseAnalytics></CourseAnalytics>
+      },
+      {
+        path: '/courses/:courseId',
+        element: <CourseDetails></CourseDetails>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)